src/simplewallet/simplewallet.cpp

```diff
@@ -3644,7 +3655,7 @@ bool simple_wallet::print_ring_members(const std::vector<tools::wallet2::pending
       {
         if (spent_key_txid[i] == spent_key_txid[j])
           are_keys_from_same_tx = true;
-        if (std::abs(spent_key_height[i] - spent_key_height[j]) < 5)
+        if (std::abs((int64_t)(spent_key_height[i] - spent_key_height[j])) < (int64_t)5)
```

Comment:
Shouldn't this be `(int64_t)spent_key_height[i] - (int64_t)spent_key_height[j]` instead?